docs/token_transferring/neonpass_usage.md

```diff
@@ -15,27 +15,27 @@ import image2 from '@site/static/img/doc-images/neonpass/transfer.png';
 - Transfer tokens to and from Solana and Neon networks with our [intuitive UI](https://neonpass.live/)
 - No wrapping
 
-
 ## Overview
+
 This guide demonstrates how to transfer tokens between Solana and Neon EVM using the [NeonPass](https://neonpass.live/) UI. The same tooling is available as an [npm package](/docs/developing/integrate/neon_transfer).
 
 ## Prerequisites
 
 - EVM-compatible wallet (e.g. Atomic Wallet, MetaMask, etc.)
-- Solana-compatible wallet (e.g. Phantom, Solfare, etc.) 
+- Solana-compatible wallet (e.g. Phantom, Solfare, etc.)
 - Token balance to pay gas fees
 
 ## Fees
 
-You can transfer tokens in either direction, but each transaction incurs a gas fee in either NEON or SOL; you decide. 
+You can transfer tokens in either direction, but each transaction incurs a gas fee in either NEON or SOL.
 
-> Under the hood, moving tokens from Neon EVM to Solana requires two transactions: one for Neon EVM (which requires a fee in NEON) and another for Solana (which requires a fee in SOL). However, NeonPass consolidates these payments, which may be made in the token of your choosing.
+> Under the hood, moving tokens from Neon EVM to Solana or vice versa requires two transactions: one for Neon EVM (which requires a fee in NEON) and another for Solana (which requires a fee in SOL). However, NeonPass consolidates these payments. Transferring tokens from Neon EVM to Solana requires the tansaction fees to be paid in NEON; whereas transferring tokens from Solana to Neon EVM requires the transaction fees to be paid in either NEON or SOL (depending on user's choice of token).
 
 <!-- Retiring this, believe it is out dated in terms of payment
 
 For example, moving tokens from Neon EVM to Solana requires two transactions, one for Neon EVM (which requires a fee in NEON) and another for Solana (which requires a fee in SOL). -->
 
-<!-- is this no longer true -- rather than account required, just wallet required? 
+<!-- is this no longer true -- rather than account required, just wallet required?
```

Comment:
@sukanyaparashar please can you verify whether line 39 is still necessary, please

Reply:
@m4sterbunny , we can have multiple accounts in a wallet. We can just say about having Ethereum-compatible wallets and Solana-compatible wallets installed in the browser.

Reply:
Please execute your idea so that it can be reviewed